### PR TITLE
Add camel-aws2-step-functions wrapper

### PR DESCRIPTION
**Pull request by f2par0**
camel-aws2-step-functions wrapper

### CODE REVIEW

**Comment by lopushen** [comment]
reviewing

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -427,6 +427,12 @@
         <bundle dependency='true'>wrap:mvn:software.amazon.awssdk/sqs/${aws-java-sdk2-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-aws2-sqs/${project.version}</bundle>
     </feature>
+    <feature name='camel-aws2-step-functions' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version="${aws-java-sdk2-version}">awssdk</feature>
```
Comment (lopushen) [inline features/src/main/feature/camel-features.xml:432]: I think this is ok, just to mention, AWS SDK versions are
2.24.0 in dependency tree
2.25.15 in camel-karaf

**Review by lopushen — APPROVED** [review]
no conflict ✔️
compilation works ✔️
no unnecessary usage of wrap protocol ✔️
no unnecessary dependencies ✔️
no hardcoded version if it's in the parent pom ✔️
feature is placed in alphabetical order in the file camel-features.xml ✔️
usage of dependent features (netty, jackson, jakarta ...) ✔️

LGTM

File: features/src/main/feature/camel-features.xml (hunk 1 above)
Comment (f2par0) [reply features/src/main/feature/camel-features.xml:432]: did you update your camel to branch camel-4.5.x ?

Comment (lopushen) [reply features/src/main/feature/camel-features.xml:432]: did not switch to it, was looking at 4.4.x, you are right, the versions correspond. Please disregard previous comment

**Review by essobedo — APPROVED** [review]

**Pull request merged**